examples/init_repo_with_deploy_keys/main.tf

```diff
@@ -0,0 +1,50 @@
+variable "github_token" {
+  type    = string
+  default = ""
+}
+
+variable "github_owner" {
+  type    = string
+  default = ""
+}
+
+provider "github" {
+  token = var.github_token
+  owner = var.github_owner
+}
+
+# This will create `terraform-example-test-webhooks` repository
+module "example" {
+  source = "../../"
+
+  namespace   = "terraform"
+  tenant      = "example"
+  name        = "test-deploy-keys"
+  description = "Terraform module example github repository"
+
+  visibility = "private"
+
+  gitignore_template = "Terraform"
+  license_template   = "mit"
+
+  vulnerability_alerts = true
+
+  default_branch_protection_enabled = false
+
+  deploy_keys = [
+    {
+      title     = "read-only"
+      key       = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAAAgQCnxrhTA8hy5xje/++8BfvbhuKYCWD9FZqNs99BXkur/2GOOw8JX3ClRPnxmFSP3wUekarv0FgIiPrUhE4g4ea1CCoqC3HhZmfoG+VsvjhJnn+LjhcVIh6MvlQ42ZixJCdH3vmLm13BVZMzFfhW1v6pYfm3hkjU7UQvMHUGiLEpvw=="
```

Comment:
In a future iteration, it'd be great to enforce a key-length... although I'm not sure we can or should do this, here. Probably in some other GitHub settings.

Reply:
I think you cannot enforce it on GitHub side. But we can add some validation on top for this input paramters.